diff --git a/setup.py b/setup.py
index 9a1d26b..5599e66 100644
--- a/setup.py
+++ b/setup.py
@@ -83,12 +83,8 @@ def get_config_posix(options=None):
 
 
 def get_config_win32(options):
-    client = "mariadbclient"
-    connector = os.environ.get("MYSQLCLIENT_CONNECTOR", options.get("connector"))
-    if not connector:
-        connector = os.path.join(
-            os.environ["ProgramFiles"], "MariaDB", "MariaDB Connector C"
-        )
+    client = "libmysqlclient"
+    connector = os.environ.get("LIBRARY_PREFIX")
 
     extra_objects = []
 
